spec/rspec/core/resources/a_custom_formatter.rb

```diff
@@ -0,0 +1,22 @@
+module Custom
+  class AGeneralFormatter < RSpec::Core::Formatters::BaseFormatter
+    RSpec::Core::Formatters.register self, :message
```

Comment:
If you remove this as above this isn't relevant, but this leaks into the rest of the spec suite and given it's not a real formatter shouldn't really be done, nor is it needed for your specs.